.buildkite/publish/run.sh

```diff
@@ -6,19 +6,21 @@ npm i --silent -g pnpm@6 --unsafe-perm # TODO: is this unsafe-perm needed?
 
 pnpm i
 
-pnpm run lint
-
 node -v
 npm -v
 
 pnpm run setup
 
 # TODO: why is this necessary?
-cd packages/integration-tests
-pnpm i sqlite3@5.0.2 --unsafe-perm --reporter=silent
-cd ../..
+# cd packages/integration-tests
+# pnpm i sqlite3@5.0.2 --unsafe-perm --reporter=silent
+# cd ../..
+
+# New client test suite
+pnpm run test:functional --filter "@prisma/client"
```

Comment:
Let's run it first, since it's where we add new tests, if it fails faster it's a better feedback loop!